tests/indexlr.cpp

```diff
@@ -92,16 +92,19 @@ main()
     }
   }
 
+  ss << std::endl;
+  ss2 << std::endl;
+
   TEST_ASSERT_EQ(ss.str(), correct_output);
   TEST_ASSERT_EQ(ss2.str(), correct_output2);
 
   std::cerr << "Testing with Bloom filters" << std::endl;
   btllib::BloomFilter filter_in_bf(1024 * 1024 * 32, 1);
   btllib::BloomFilter filter_out_bf(1024 * 1024 * 32, 1);
 
-  std::vector<uint64_t> filter_in_hashes = { 430447521414431149ULL,
-                                             3146270839399521840ULL,
-                                             161808173335193798ULL };
+  std::vector<uint64_t> filter_in_hashes = { 1315163655624994337ULL,
+                                             4261937130627716230ULL,
+                                             6055236352205909654ULL };
   std::vector<uint64_t> filter_out_hashes = { 1672947938795563804ULL,
                                               2858314356342515870ULL,
                                               1712341822067595113ULL };
```

Comment:
I think these need to be updated too. I imagine right now these aren't ever produced so the test doesn't fail, but they should be hashes that are produced by some k-mers in the input, but indexlr should ignore them when considering k-mers for minimizers.

Reply:
Ideally, we'd have a test that makes sure that the hash values are present in some k-mers, but not picked as minimizers.